src/cpu/sys/windows/cpu_times.rs

```diff
@@ -0,0 +1,88 @@
+use crate::cpu::CpuTimes;
```

Comment:
Imports should be organized into these sections and in this order: std, dependencies, and crate. So these crate dependencies should be moved to line 19. There's a few cases of this across the PR.